packages/block-editor/src/layouts/flow.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	Button,
+	__experimentalUseCustomUnits as useCustomUnits,
+	__experimentalUnitControl as UnitControl,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { Icon, positionCenter, stretchWide } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import useSetting from '../components/use-setting';
+import { appendSelectors } from './utils';
+
+export default {
+	name: 'default',
+
+	label: __( 'Flow' ),
+
+	edit: function LayoutDefaultEdit( { layout, onChange } ) {
+		const { wideSize, contentSize } = layout;
+		const units = useCustomUnits( {
+			availableUnits: useSetting( 'spacing.units' ) || [
+				'%',
+				'px',
+				'em',
+				'rem',
+				'vw',
+			],
+		} );
+
+		return (
+			<>
+				<div className="block-editor-hooks__layout-controls">
+					<div className="block-editor-hooks__layout-controls-unit">
+						<UnitControl
+							label={ __( 'Content' ) }
+							labelPosition="top"
+							__unstableInputWidth="80px"
+							value={ contentSize || wideSize || '' }
+							onChange={ ( nextWidth ) => {
+								nextWidth =
+									0 > parseFloat( nextWidth )
+										? '0'
+										: nextWidth;
+								onChange( {
+									...layout,
+									contentSize: nextWidth,
+								} );
+							} }
+							units={ units }
+						/>
+						<Icon icon={ positionCenter } />
+					</div>
+					<div className="block-editor-hooks__layout-controls-unit">
+						<UnitControl
+							label={ __( 'Wide' ) }
+							labelPosition="top"
+							__unstableInputWidth="80px"
+							value={ wideSize || contentSize || '' }
+							onChange={ ( nextWidth ) => {
+								nextWidth =
```

Comment:
This is just the existing code I didn't touch it. It seems if there's something wrong here it's `UnitControl` passing `px` when the value is empty instead of just ''